### PR TITLE
feat(mrf): disable unsupported features

### PR DESCRIPTION
**Pull request by KenLSM**
## Problem

The users should not be able to use the features that do not work for the beta release
Closes FRM-1663

## Solution

**Breaking Changes** 

- No - this PR is backwards compatible  

### Attachments are disabled
disable attachments

### Mobile OTP is disabled
disable mobile otp

### Email OTP + Confirmation is disabled
email-mrf

##### New feature tests
Admin MRF Forms - Unsupported fields are disabled
- [ ] As an Admin create a non-MRF form
- [ ] Ensure that Attachment Field is disabled
  - [ ] Ensure that Attachment Field **cannot** be added as a form field
- [ ] Ensure that on Mobile Field, the OTP toggle is disabled
- [ ] Ensure that on Email Field, the OTP toggle is disabled
- [ ] Ensure that on Email Field, the OTP and email confirmation toggles are disabled

Respondent MRF Forms - Feedback section is not shown
- [ ] As a respondent, submit an MRF form
- [ ] Ensure that form feedback section is not shown

##### Regression
Non-MRF Forms - fields that are disabled on MRF should not affect non-MRF forms
- [ ] As an Admin create a non-MRF form
- [ ] Ensure that Attachment Field is **not** disabled
- [ ] Ensure that Attachment Field **can** be added as a form field 
- [ ] Ensure that on Mobile Field, the OTP toggle is not disabled
- [ ] Ensure that on Email Field, the OTP and email confirmation toggles are **not** disabled

Respondent Non-MRF Forms - Feedback section is shown
- [ ] As a respondent, submit a form
- [ ] Ensure that form feedback section is shown

### CODE REVIEW

File: frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx [hunk 1]
```diff
@@ -72,14 +72,18 @@ export const FormEndPageContainer = (): JSX.Element | null => {
       />
     )
   }
+
+  // Feedback is not supported on MRF
+  const isHideFeedback = form.responseMode === FormResponseMode.Multirespondent
+  console.log(isHideFeedback)
```
Comment (justynoh) [inline frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx]: redundant console log. Also, `isFeedbackHidden` maybe? `isHideFeedback` reads weirdly ahaha

**Review by justynoh — COMMENTED** [review]
eh sorry ah. one more HAHA can we disable email confirmations too? in the EditEmail tab. thank you haha

File: frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx (hunk 1 above)
Comment (KenLSM) [reply frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx]: Great catch -- thought I've removed the comments earlier 🤦 .

cd19eb895

**Review by justynoh — APPROVED** [review]
thank you sir! lgtm

File: frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx [hunk 2]
```diff
@@ -72,14 +72,18 @@ export const FormEndPageContainer = (): JSX.Element | null => {
...
+    form.responseMode === FormResponseMode.Multirespondent
   return (
     <Box py={{ base: '1.5rem', md: '2.5rem' }} w="100%">
       <FormEndPage
         colorTheme={form.startPage.colorTheme}
         submissionData={submissionData}
         formTitle={form.title}
         endPage={form.endPage}
-        isFeedbackSubmitted={isFeedbackSubmitted}
+        hideFeedbackSection={isFeedbackSubmitted || isFeedbackHidden}
```
Comment (justynoh) [inline frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx]: this prop naming change caused a visual regression on storybook, i think. May need to trouble you to update the `FormEndPage`stories to use this new prop!

_Also separate supernit while I'm relooking this:_
Also now i'm wondering if it makes sense to call the prop `isFeedbackSectionHidden` for consistency with other boolean props, and then move the computation into the const above.

```typescript
const isFeedbackSectionHidden = form.responseMode === FormResponseMode.Multirespondent || isFeedbackSubmitted

return (
  //...
  <FormEndPage 
    //...
    isFeedbackSectionHidden={isFeedbackSectionHidden}
    />
  //...
  )

Comment (KenLSM) [reply frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx]: 77eabe1

**Pull request merged**